src/components/book.js

```diff
@@ -12,13 +15,27 @@ const Book = (prop) => {
   };
 
   return (
-    <div>
-      <h3>{title}</h3>
-      <p>{author}</p>
-      <button type="button" onClick={clickHandler}>
-        remove
-      </button>
-    </div>
+    <section className="book">
+      <div>
+        <h4>{category}</h4>
+        <h3>{title}</h3>
+        <p className="blueText">{author}</p>
+        <ul className="bookButtonList">
+          <li className="blueText bookButton">Comments</li>
+          <li>
+            <button
+              onClick={clickHandler}
+              type="button"
+              className="blueText removeButton bookButton"
+            >
+              Remove
+            </button>
+          </li>
+          <li className="blueText bookButton">Edit</li>
+        </ul>
+      </div>
+      <img src={bookStatus} alt="bookStatus" id="bookImage" />
+    </section>
```

Comment:
- [ ] Please kindly make sure that this part of the Book component is built out with HTML elements and properly styled according to the design instead of using screenshot images.

![image](https://user-images.githubusercontent.com/43995830/179356323-249da421-fa73-4633-8cb5-12e928990c48.png)